Tests/SessionTests.swift

```diff
@@ -858,9 +858,9 @@ final class SessionTestCase: BaseTestCase {
 
         // Then
         for request in requests {
-            if let error = request.error?.asAFError {
-                XCTAssertTrue(error.isRequestAdaptationError)
-                XCTAssertEqual(error.underlyingError?.asAFError?.urlConvertible as? String, "")
+            if let error = request.error {
+                XCTAssertTrue(error.isInvalidURLError)
```

Comment:
I think this is another case where we *should* wrap an `AFError` in an `AFError`, as we want users to know it's an adaptation error and why, both of which can be `AFError` instances.

Reply:
Reverted.